src/screens/SettingsScreen.tsx

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
-import { Settings } from '../components';
+import { Settings, SignoutButton } from '../components';
 import { Text } from 'react-native';
+import Constants from 'expo-constants';
+import { Container } from '../components/styling/SettinngsScreenStyles';
+import { from } from 'zen-observable';
```

Comment:
don't think this is being used

Also the settings styling component file is not grammatically correct. Lets fix that.